Makefile

```diff
@@ -125,6 +127,9 @@ ifneq (,$(findstring $(ARCH_TAG),x86 x64 arm64))
   CXXFLAGS += -momit-leaf-frame-pointer
 endif
 
+ifeq ($(OS_TAG),linux)
+  CXXFLAGS += -ggdb
```

Comment:
There is already `ifeq ($(OS),Darwin)` branch above that sets CXXFLAGS depending on a platform, should we move it there?